pytest_invenio/fixtures.py

```diff
@@ -589,3 +591,70 @@ def create_bucket_from_dir(source_dir, location_obj=None):
         db.session.commit()
         return bucket_obj
     return create_bucket_from_dir
+
+
+@pytest.fixture(scope='module')
+def inv_cache(appctx):
```

Comment:
Can the fixtures be named without the ``inv_`` prefix?

Reply:
Pytest already provides a `cache`, so we can't use that. But I am open to better suggestions, don't really like the `inv_` prefix.